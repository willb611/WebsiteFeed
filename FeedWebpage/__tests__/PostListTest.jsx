import React from 'react';
import renderer from 'react-test-renderer';
import PostList from '../Scripts/PostList.jsx';

describe('Welcome (Snapshot)', () => {
    it('Welcome renders PostList', () => {
        const someData = [];
        const component = renderer.create(<PostList data={someData}/>);
        const json = component.toJSON();
        expect(json).toMatchSnapshot();
    });
});
