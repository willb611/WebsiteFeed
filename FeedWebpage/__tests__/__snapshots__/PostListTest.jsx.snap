exports[`Welcome (Snapshot) Welcome renders PostList 1`] = `
<ul
  className="postList" />
`;
